ga4gh/backends.py

```diff
@@ -411,20 +416,20 @@ def searchVariantSets(self, request):
         if request.pageSize is None:
             request.pageSize = 100
         response = protocol.GASearchVariantSetsResponse()
-        j = 0
+        currentIndex = 0
         if request.pageToken is not None:
-            j = int(request.pageToken)
-        v = []
-        while j < len(self._variantSetIds) and len(v) < request.pageSize:
-            vs = protocol.GAVariantSet()
-            vs.id = self._variantSetIds[j]
-            vs.datasetId = "NotImplemented"
-            vs.metadata = self._variantSets[vs.id].getMetadata()
-            v.append(vs)
-            j += 1
-        if j < len(self._variantSetIds):
-            response.nextPageToken = j
-        response.variantSets = v
+            currentIndex = int(request.pageToken)
+        variantSetList = []
+        while currentIndex < len(self._variantSetIds) and len(variantSetList) < request.pageSize:
```

Comment:
pretty sure you'd want to use slices here. Am I missing something? Something like:

```python
start = int(request.pageToken)
end = start + request.pageSize
variantSetIdsInPage = self._variantSetIds[start:end]
for variantSetId in variantSetIdsInPage:
  ...
response.nextPageToken = start + len(variantSetList)
```